.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  rebase-strategy: disabled
```

Comment:
It's not a setting we use in other repos, is this intentional?

Reply:
copy pasta, removed this to be consistent.